### DISCUSSION TITLE
Timeline by days instead of years?

### DISCUSSION THREAD

**Discussion by mjeleon** [CollectionBuilder/collectionbuilder.github.io#124] [Q&A] [answered]
Is it possible to use the timeline to group entries by specific dates instead of years? I have a collection of items that all take place within the same year, and would love to group and list them by date instead if possible. I tried following the instructions to [repurpose the timeline](https://collectionbuilder.github.io/cb-docs/docs/advanced/repurposetimeline/)- but the instructions did not match the code I was looking at- not sure if that's because the instructions are outdated, or if they were not meant for CollectionBuilder GH, which is what I'm using. Any ideas?

**Comment by owikle** [comment, marked as answer]
@mjeleon the easiest way to do this is to go to _layouts > timeline.html. Change line 17 from this:

```
{%- capture clean-days -%}{% for date in raw-dates %}{% if date contains "-" %}{{ date | strip | split: "-" | first }}{% elsif date contains "/" %}{{ date | strip | split: "/" | last }}{% else %}{{ date | strip }}{% endif %}{% unless forloop.last %};{% endunless %}{%- endfor -%}{%- endcapture -%}
```

to this:

```
{%- capture clean-days -%}{% for date in raw-dates %}{{ date | strip }}{% unless forloop.last %};{% endunless %}{%- endfor -%}{%- endcapture -%}
```

Commit the change and you should see the items in your timeline now grouped by day instead of year. Good luck!

**Reply by mjeleon** [reply]
Worked like a charm, thank you so much!
